www/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "compe-plus-www",
+    "description": "Front end for CompE+",
+    "version": "0.1.0",
```

Comment:
Maybe bump the api version back to `0.1.0` too. Let's save `1.0.0` for when we have an MVP.

This opens a bigger question of whether we want to be updating these versions every time we change things. Not really necessary I guess b/c we won't be publishing packages but is kind of idiomatic.

Reply:
wait, was it `1.0.0`? it says `0.1.0` for me right now. I can't tell if I'm tripping loool

Reply:
oh you said **api version** lol ok changing that now